docs/tutorials/gettingstartedexample/gse-tap-common.md

```diff
@@ -0,0 +1,351 @@
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+---
+
+## Prerequisites
+A supported interface module that includes both serial input and serial output channels (tap module) and a second supported interface module that includes serial input channels (acq module) on a PXI system running Windows.
+
+A camera supported by the getting started example configuration scripts (i.e. Leopard Imaging IMX490) or a third supported interface module that includes serial output channels (gen module).
+
+| **Interface Module**                        | **Camera**                                                                                                                            |
+|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| PXIe-1486 (8 In - 954 Deserializer)         | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
+| PXIe-1486 (4 In 4 Out - 953/954 SerDes)     | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
+| PXIe-1487 (8 I - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes) | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+
+> Note: The tap and acq modules used in this tutorial must have matching model numbers (i.e. 1486).
+
+This tutorial is written for users who understand how to perform a basic tap acquisition with PXIe-148X GMSL or FPD-Link interface modules. At a minimum it is required that you complete the [Initial Hardware Setup](gse-tap-basic.md#initial-hardware-setup) and [Initial Software Setup](gse-tap-basic.md#initial-software-setup) from the PXIe-148X Getting Started Example - Basic Tap Tutorial before starting this tutorial.
+
+> Note: Tutorials in this document use one tap LabVIEW project and one acquisition LabVIEW project as well as a camera or a generation LabVIEW project to provide the image data to the tap module serial input. All tutorial steps in this document reference the Tap Example VI unless otherwise specified and all input control values not specified should be left as the default value.
+
+## Running a Generation-Tap-Acquisition Getting Started Example Combination
+This tutorial uses a gen module, tap module and acq module to demonstrate basic tap capabilities using a gen module in place of a camera. The sequence for this tutorial is configure generation example, configure tap example, configure acquisition example, start acquisition example, start tap example, start generation example, wait for generation example to complete, stop acquisition example, stop tap example.
```

Comment:
Consider some kind of list to make this more readable:

 The sequence for this tutorial is:
1. **Configure:** configure gen example, configure tap example, configure acq example
2. **Start:** start acq example, start tap example, start gen example
3. **Wait:** wait for gen example to complete (stops automatically)
4. **Stop:** stop acq example, stop tap example.

Reply:
I made the suggested update.  Since this precedes a numbered list I had to put the sequence in a blockquote section. I confirmed that it renders properly on the github.io page.